infrastructure/staging/terraform.tfvars

```diff
@@ -0,0 +1,17 @@
+region = "us-east-1"
+
+saml_role = "TFState"
+
+app = "phoenix-starter"
+
+env = "staging"
+
+team = "newaperio"
+
+customer = "test"
+
+tags = {
+  Terraform = "true"
```

Comment:
Other tags are lowercased, should this? Or vice-versa? Just for consistency.

Reply:
tags are case insensitive, but I can capitalize them across the board